Integrations/Import.lua

```diff
@@ -61,17 +63,34 @@ local function ParseImportData(self)
                 }
             end
             rosterMap[roster.uid].players[#rosterMap[roster.uid].players+1] = player.guid
-            if player.dkp ~= 0 then
+
+            if player.dkp and player.dkp ~= 0 then
```

Comment:
If we are moving from `dkp` nomenclature to `points` this one is redundant and shouldn't be used here as `points` would cover DKP

Reply:
I wasn't able to test the export functionality with a dkp setup, I agree with the sentiment but was unable to confirm the ability to remove this field.  Additionally this version maintains compatibility with old exports while only including an additional null check